lib/rules/jsx-no-target-blank.js

```diff
@@ -103,7 +104,47 @@ module.exports = {
           context.report({
             node,
             message: 'Using target="_blank" without rel="noreferrer" '
-              + 'is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener'
+              + 'is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener',
+            fix(fixer) {
+              const relAttribute = node.parent.attributes.find((attr) => attr.name.name === 'rel');
+              if (!relAttribute) {
+                return fixer.insertTextAfter(node, ' rel="noreferrer"');
+              }
+
+              if (!relAttribute.value) {
```

Comment:
can you add test cases for `rel=""`, `rel={false}`, `rel={0}`, etc?

Reply:
sure

Reply:
already existed for `""`. So I add for false, 0, and null